app/workers/activitypub/reply_distribution_worker.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class ActivityPub::ReplyDistributionWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'push'
+
+  def perform(status_id)
+    @status  = Status.find(status_id)
+    @account = @status.thread.account
+
+    return if skip_distribution?
+
+    ActivityPub::DeliveryWorker.push_bulk(inboxes) do |inbox_url|
+      [signed_payload, @account.id, inbox_url]
```

Comment:
When alice replies bob, json includes alice as `actor` and bob as `account_username`, then signed by bob's key. In this case, receiver instance treats the status is authored by bob instead of alice.

Reply:
You're right, I messed it up. We need status.thread.account's followers, but we need status.account's signature.